config/jobs/kubernetes-sigs/cluster-api-provider-ibmcloud/cluster-api-provider-ibmcom-presubmits.yaml

```diff
@@ -108,3 +108,35 @@ presubmits:
     annotations:
       testgrid-dashboards: sig-cluster-lifecycle-cluster-api-provider-ibmcloud
       testgrid-tab-name: pr-verify
+  - name: pull-cluster-api-provider-ibmcloud-coverage
+    always_run: true
+    optional: true
```

Comment:
any specific reason for making this as an optional?

Reply:
Removed. this should not be optional.